identity-wallet/src/state/reducers/authorization.rs

```diff
@@ -261,19 +279,33 @@ pub async fn handle_oid4vp_authorization_request(state: AppState, action: Action
                 ConnectionRequest::SIOPv2(_) => unreachable!(),
             };
 
+        let mut history_credentials = Vec::new();
+
         let verifiable_credentials: Vec<Credential<JwtVcJson>> = stronghold_manager
             .values()
             .map_err(StrongholdValuesError)?
             .unwrap()
             .iter()
-            .filter_map(
-                |verifiable_credential_record| match &verifiable_credential_record.verifiable_credential {
+            .filter_map(|verifiable_credential_record| {
+                let share_cred = match &verifiable_credential_record.verifiable_credential {
```

Comment:
Preferably please follow the general naming convention by avoiding abbreviations :)